src/resmom/vnode_storage.c

```diff
@@ -152,25 +152,25 @@ find_vmapent_byID(void *ctx, const char *vnid)
  * @param[in] data - information about vnode
  *
  * @return 	int
- * @retval 	0	Success
- * @retval 	1	Failure
+ * @retval 	1	Sucess
+ * @retval 	0	Failure
```

Comment:
As part of PR #1737, I had reverted the error code for add_vmapent_byID() based on the function header. Based on the call path new_vnid() -> add_mominfo() -> add_vmapent_byID(), the return values are correct. So reverted those changes (kept the refactored code for add_vmapent_byID() as is) and updated the function header to reflect correct success and failure values. 
In case of failure we are returning zero in add_vmapent_byID()

Reply:
Do you think we should update add_mominfo header too ? Because it says it return 0 for success, which may contradict now ?